content/applications/finance/documents.rst

```diff
@@ -0,0 +1,126 @@
+:show-content:
```

Comment:
This page is not a parent page (no `toctree` in it) so `:show-content:` is useless here.